techniques/system/distributePolicy/1.0/rsyslogConf.st

```diff
@@ -18,9 +18,17 @@
 
 # Configure rsyslog on the root/relay servers
 
-bundle agent install_rsyslogd {
-
+bundle agent install_rsyslogd
+{
   vars:
+&if(INITIAL)&
+    # This will only read the first line if several match
+    "rsyslog_port_defined"     int => getfields("rudder.syslog.port\s*=\s*[0-9]+\s*","/opt/rudder/etc/rudder-web.properties","=","rsyslog_port");
+
+    # Use standard rsyslog port number by default if not set in rudder-web.properties
+    rsyslog_port_not_found::
+      "rsyslog_port[2]"    string => "&DEFAULT_SYSLOGPORT&";
```

Comment:
Why use a StringTemplate variable for this? It's only used once, and it's a well known value, we could just write 514 like the initial-promises used to. It seems overly complex to me but maybe I'm missing something?